ner_v2/detectors/numeral/utils.py

```diff
@@ -0,0 +1,108 @@
+import re
+
+
+def get_number_from_number_word(text, number_word_dict):
```

Comment:
Here are a few cases breaking 
```
print(get_number_from_number_word('three thousand two hundred and ten', number_words_map))
print(get_number_from_number_word('ninety nine thousand nine hundred and ninety nine', number_words_map))
print(get_number_from_number_word('three hundred thousand', number_words_map))
print(get_number_from_number_word('hundred thousand', number_words_map))
print(get_number_from_number_word('twenty three', number_words_map))
```

```
([3211], ['three thousand two hundred and ten'])
([100000], ['ninety nine thousand nine hundred and ninety nine'])
([1300], ['three hundred thousand'])
([1100], ['hundred thousand'])
([23], ['twenty three'])
```

In the first two cases, the inclusion of `and` with `scale=1, increment=0` adds 1 to the result.

Some of these I know we are not handling, we should document these limitations somewhere
```
1. Cannot detect decimals without the integer part. E.g. .25, .5, etc
2. Cannot detect one with "a/an". E.g. I want an apple 
3. Detects wrong for [2-9][1-9] (21 to 99 except multiples of ten) in words E.g. twenty five, ninety nine
4. Detects wrong for multiple scales mentioned consecutively E.g. three hundred thousand, hundred thousand
```